packages/suite-desktop/src-electron/preload.ts

```diff
@@ -51,12 +51,17 @@ contextBridge.exposeInMainWorld('desktopApi', {
             ipcRenderer.once(channel, (_, ...args) => func(...args));
         }
     },
+    // todo: off does not even exist in electron documentation anymore
+    // https://www.electronjs.org/docs/api/ipc-renderer
     off: (channel: string, func: (...args: any[]) => any) => {
+        ipcRenderer.removeListener(channel, func);
         if (validChannels.includes(channel)) {
             ipcRenderer.off(channel, (_, ...args) => func(...args));
         }
     },
-
+    removeAllListeners: (channel: string) => {
+        ipcRenderer.removeAllListeners(channel);
+    },
```

Comment:
implement this:
https://github.com/trezor/trezor-suite/pull/3175/files#diff-7a95c75c77571983a267b8413959e765e96268f9506d139a165c1253669818d7